contrib/automation_tests/test_cases/connection_window.py

```diff
@@ -34,7 +36,7 @@ class LoadCapture(E2ETestCase):
     It raises an exception if the capture does not exists. Also waits until the capture is loaded.
     """
 
-    def _execute(self, capture_file_path: str):
+    def _execute(self, absolute_capture_file_path=None, relative_capture_file_path=None):
```

Comment:
Isn't there a way to tell whether a path is relative or absolute from the path itself?

Reply:
`os.path.isabs()`

Reply:
Done.

Reply:
You should also adjust the comment of the class to point out a relative path is accepted as well. Also mention that it's relative to Orbit.exe, I think this is not obvious (I would have expected it to be relative to our automation test root dir).

Reply:
Looking at the path used in this test, does it even make sense to make it relative to Orbit.exe?

Reply:
Made it relative to the `automation_tests` directory and adjusted the comment.